app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class= "comments-list">
+  <% if comments.any? %>
+    <%comments.reverse_each do |comment|%>
+      <div class="comment">
+        <li><%= comment.content %></li>
+        <% if owned?(comment)%>
+        	<% if comment.commentable_type == "Answer" %>
```

Comment:
It might help readability to have a helper to create these links instead of if/else blocks.